Uplift/Views/GymDetail/Facilities/EquipmentListItemCell.swift

```diff
@@ -19,6 +19,10 @@ class EquipmentListItemCell: ListItemCollectionViewCell<EquipmentCategory> {
     private let equipmentList = UITextView()
     private let quantityList = UITextView()
 
+    // MARK: - Private layout vars
+    private let listsSpacing: CGFloat = 12
+    private var equipmentListTrailingConstraint: Constraint?
```

Comment:
why do we need this?

Reply:
I think that was from an old merge, I'll delete it here